internal/convert/convert.go

```diff
@@ -254,3 +309,99 @@ func GtfsRealtimeExtension(in *api.GtfsRealtimeOptions) (extensions.Extension, e
 		return nil, fmt.Errorf("unknown extension %s", in.Extension)
 	}
 }
+
+func NullVehicleCurrentStatus(currentStatus *gtfs.CurrentStatus) pgtype.Text {
+	if currentStatus == nil {
+		return pgtype.Text{}
+	}
+
+	switch *currentStatus {
+	case 0:
+		return pgtype.Text{
+			Valid:  true,
+			String: "INCOMING_AT",
+		}
+	case 1:
+		return pgtype.Text{
+			Valid:  true,
+			String: "STOPPED_AT",
+		}
+	case 2:
+		return pgtype.Text{
+			Valid:  true,
+			String: "IN_TRANSIT_TO",
+		}
+	default:
+		return pgtype.Text{}
+	}
+}
+
+func NullCongestionLevel(congestionLevel *gtfs.CongestionLevel) string {
+	if congestionLevel == nil {
+		return CongestionLevel(0)
+	}
+	return CongestionLevel(*congestionLevel)
+}
+
+func CongestionLevel(congestionLevel gtfs.CongestionLevel) string {
+	switch congestionLevel {
+	case 0:
+		return "UNKNOWN_CONGESTION_LEVEL"
+	case 1:
+		return "RUNNING_SMOOTHLY"
+	case 2:
+		return "STOP_AND_GO"
+	case 3:
+		return "CONGESTION"
+	case 4:
+		return "SEVERE_CONGESTION"
+	}
+
+	return "UNKNOWN_CONGESTION_LEVEL"
+}
+
+func NullOccupancyStatus(occupancyStatus *gtfs.OccupancyStatus) pgtype.Text {
+	if occupancyStatus == nil {
+		return pgtype.Text{}
+	}
+
+	switch *occupancyStatus {
+	case 0:
```

Comment:
I was going to suggest using names rather than numbers, but I think the problem is that the names aren't available in the `gtfs` namespace? Could we just add a TODO to figure out using the names? Also I wonder if `occupancyStatus.String()` exists and gives the right answer for all of these?

Reply:
`.String()` works, will replace magic numbers with this!